pkg/requests/ids.go

```diff
@@ -0,0 +1,51 @@
+package requests
+
+import "regexp"
+
+var idURLMap = map[string]string{
+	"txn":   "/v1/balance_transactions/",
+	"ch":    "/v1/charges/",
+	"cus":   "/v1/customers/",
+	"dp":    "/v1/disputes/",
+	"evt":   "/v1/events/",
+	"file":  "/v1/files/",
+	"link":  "/v1/file_links/",
+	"pi":    "/v1/payment_intents/",
+	"seti":  "/v1/setup_intents/",
+	"po":    "/v1/payouts/",
+	"prod":  "/v1/products/",
+	"re":    "/v1/refunds/",
+	"tok":   "/v1/tokens/",
+	"pm":    "/v1/payment_methods/",
+	"src":   "/v1/sources/",
+	"cs":    "/v1/checkout/sessions/",
+	"cn":    "/v1/credit_notes/",
+	"txi":   "/v1/tax_ids/",
+	"in":    "/v1/invoices/",
+	"ii":    "/v1/invoice_items/",
+	"sub":   "/v1/subscriptions/",
+	"si":    "/v1/subscription_items/",
+	"txr":   "/v1/tax_rates/",
+	"acct":  "/v1/accounts/",
+	"fee":   "/v1/application_fees/",
+	"tu":    "/v1/topups/",
+	"tr":    "/v1/transfers/",
+	"issfr": "/v1/radar/early_fraud_warnings/",
+	"prv":   "/v1/reviews/",
+	"rsl":   "/v1/radar/value_lists/",
+	"rsli":  "/v1/radar/value_list_items/",
+	"iauth": "/v1/issuing/authorizations/",
+	"ich":   "/v1/issuing/cardholders/",
+	"ic":    "/v1/issuing/cards/",
+	"idp":   "/v1/issuing/disputes/",
+	"ipi":   "/v1/issuing/transactions/",
+	"tml":   "/v1/terminal/locations/",
+	"tmr":   "/v1/terminal/readers/",
+	"or":    "/v1/orders/",
+	"orret": "/v1/order_returns/",
+	"sku":   "/v1/skus/",
+	"sqr":   "/v1/sigma/scheduled_query_runs/",
+	"we":    "/v1/webhook_endpoints/",
+}
+
+var idRegex = regexp.MustCompile("^([a-z]{2,5})_(test_|live_)?[a-zA-Z0-9]{3,}$")
```

Comment:
Do we actually use `test_|live_` in IDs?

Reply:
I think so! That's how our api ref documents checkout sessions: https://stripe.com/docs/api/checkout/sessions/retrieve

Reply:
👍